src/sentry/notifications/utils/__init__.py

```diff
@@ -127,10 +129,27 @@ def summarize_issues(
     return rv
 
 
-def get_link(group: Group, environment: str | None) -> str:
+def get_group_settings_link(
+    group: Group,
+    environment: str | None,
+    rule_details: Sequence[NotificationRuleDetails] | None = None,
+    alert_timestamp: int | None = None,
+) -> str:
+    alert_type = str(AlertRuleTriggerAction.Type.EMAIL.name).lower()
+    alert_timestamp_str = (
+        str(round(time.time() * 1000)) if not alert_timestamp else str(alert_timestamp)
+    )
```

Comment:
@scttcper Just checking that a timestamp works for you here

Reply:
yep i'd like a unix timestamp to avoid issues with encoding url parameters